cli/internal/fs/fs.go

```diff
@@ -63,8 +63,15 @@ func FileExists(filename string) bool {
 // CopyFile copies a file from 'from' to 'to', with an attempt to perform a copy & rename
 // to avoid chaos if anything goes wrong partway.
 func CopyFile(from string, to string, mode os.FileMode) error {
+	fileInfos, err := os.Lstat(from)
+	isSymlink := err == nil && fileInfos.Mode()&os.ModeSymlink == os.ModeSymlink
+
 	fromFile, err := os.Open(from)
 	if err != nil {
+		if isSymlink {
```

Comment:
Since we didn't previously error in the event of encountering a broken symlink this maintains that behavior.

We can revisit whether or not this is a good idea later, but this PR aims to maintain consistency.